haplotypecaller-gvcf-gatk4.wdl

```diff
@@ -214,7 +214,7 @@ task HaplotypeCaller {
       -I ~{input_bam} \
       -L ~{interval_list} \
       -O ~{output_filename} \
-      -contamination ~{default=0 contamination} \
+      -contamination ~{default="0" contamination} \
```

Comment:
Hi @jdidion, 
Why do you say the placeholder option must be a string? Reading the `default` subsection in the link provided shows an example of the `default` function being used in a task that is expecting a string but that doesn't seem to be a requirement for the function. Under Additional Requirements the docs says "The type of the expression must match the type of the parameter" so i would assume the default parameter should match the initial declaration of the variable type for `contamination`.

Reply:
I agree the text of the spec is not clear. "This task takes an optional `String` parameter" implies that the value of `default` must be a string, while `The type of the expression must match the type of the parameter` implies that the parameter and the expression types co-vary.

I have since updated wdlTools to allow any value for default as long as it is coercible to string. But the point is kind of moot since in the development (soon to be 2.0) version of WDL, `default` (and the other placeholder options) are eliminated in favor of something like:

`~{select_first([contamination, 0])}`

Feel free to close this issue.

Reply:
Thanks for the explanation!